lib/internal/modules/esm/loader.js

```diff
@@ -315,13 +325,20 @@ class ModuleLoader {
    * @param {string} [parentURL] The URL path of the module's parent.
    * @param {ImportAssertions} importAssertions Assertions from the import
    *                                            statement or expression.
-   * @returns {Promise<{ format: string, url: URL['href'] }>}
+   * @returns {{ format: string, url: URL['href'] }}
```

Comment:
Is this just an unrelated cleanup? (I think it’s fine if it is, but just making sure it’s not something that this PR is changing.)